### ISSUE TITLE
csv export dates shall be parseble by libreoffice

### ISSUE THREAD

**Issue by bernhardreiter** [Intevation/intelmq-fody#57]
Currently a csv export looks like
```csv
2869,Sat Jun 24 2017 12:00:00 GMT+0200 (CEST)
93,Sun Jun 25 2017 03:00:00 GMT+0200 (CEST)
```

which libreoffice (tested with 5) cannot directly import as datetime value.
It would be nice if it could.

**Comment by bernhardreiter** [comment]
https://help.libreoffice.org/Calc/Converting_Text_to_Numbers says

> The following ISO 8601 formats are converted:

[..]
> CCYY-MM-DDThh:mm

[..]
> Instead of the T date and time separator, exactly one space character may be used.

(which corresponds with the included help with Source: libreoffice
Version: 1:5.2.7-1~bpo8+1 from Debian)

So explicitely: There is no 'Z' allowed to indicate UTC.

**Comment by bernhardreiter** [comment]
Fixed with b0497ebc39f102505785749ef95df178ed1fea28.
